### PR TITLE
fix: make sure that user stays on same route when he manually refresh…

### PR DESCRIPTION
**Pull request by evans-g-crsj**
…es a page + remove unneeded code.

### CODE REVIEW

File: src/stateProviders/provideLoggedInUser.js [hunk 1]
```diff
@@ -127,24 +118,7 @@ export default provideState({
...
-              ? without(fields, 'institution', 'jobTitle')
-              : fields;
-          const profile = pick(omit(user, 'sets'), userRoleProfileFields);
-          const filledFields = Object.values(profile || {}).filter(
-            (v) => (isArray(v) && v.length) || (!isArray(v) && v),
-          );
-          const percentageFilled = filledFields.length / userRoleProfileFields.length;
-          if (state.loggedInUser && state.percentageFilled < 1 && percentageFilled >= 1) {
-            trackUserInteraction({
-              category: TRACKING_EVENTS.categories.user.profile,
```
Comment (francisl) [inline src/stateProviders/provideLoggedInUser.js:143]: its ok to remove tracking?

File: src/stateProviders/provideLoggedInUser.js [hunk 2]
```diff
@@ -71,8 +62,8 @@ export default provideState({
                 history.push(ROUTES.join);
               } else if (!user.acceptedTerms) {
                 history.push(ROUTES.termsConditions);
-              } else {
-                history.push(ROUTES.login);
+              } else if (['/', '/join', '/orcid'].includes(window.location.pathname)) {
```
Comment (francisl) [inline src/stateProviders/provideLoggedInUser.js:65]: why should it go to dashboard if you hit /join?

Comment (evans-g-crsj) [reply src/stateProviders/provideLoggedInUser.js:65]: it's legacy code (not visible in the left panel for I removed it). I think it means that if you type in /join in the url but you are already logged in then go to dashboard. These pages are hidden if logged in (except for '/')

File: src/stateProviders/provideLoggedInUser.js (hunk 1 above)
Comment (evans-g-crsj) [reply src/stateProviders/provideLoggedInUser.js:143]: In this case, yes. The concept of 'at what % the profile is completed' doesn t exist anymore.

**Pull request merged**